package.json

```diff
@@ -44,8 +44,10 @@
     "@babel/plugin-syntax-decorators": "^7.2.0",
     "@babel/plugin-syntax-jsx": "^7.2.0",
     "@babel/plugin-syntax-typescript": "^7.3.3",
-    "@storybook/addon-actions": "^4.1.13",
-    "@storybook/react": "^4.1.13",
+    "@emotion/core": "^10.0.7",
+    "@emotion/styled": "^10.0.7",
```

Comment:
This is a workaround for https://github.com/storybooks/storybook/issues/5817#issuecomment-470032349

Should be able to delete these.